docs/tutorials/build-a-dapp/pallet.md

```diff
@@ -135,18 +135,18 @@ std = [
 
 ### Configuration
 
-Every pallet has a component called `Trait` that is used for configuration. This component is a
+Every pallet has a component called `Config` that is used for configuration. This component is a
 [Rust "trait"](https://doc.rust-lang.org/book/ch10-02-traits.html); traits in Rust are similar to
 interfaces in languages such as C++, Java and Go. For now, the only thing we will configure about
-our pallet is that it will emit some Events. The `Trait` interface is another topic that will be
+our pallet is that it will emit some Events. The `Config` interface is another topic that will be
 covered in greater depth in the next tutorial, the [Add a Pallet](../add-a-pallet) tutorial.  The 
 following code will need to be added to your `pallets/template/src/lib.rs` file:
 
 ```rust
 /// Configure the pallet by specifying the parameters and types on which it depends.
-pub trait Trait: frame_system::Trait {
-    /// Because this pallet emits events, it depends on the runtime's definition of an event.
-    type Event: From<Event<Self>> + Into<<Self as frame_system::Trait>::Event>;
+pub trait Config: frame_system::Config {
+	  /// Because this pallet emits events, it depends on the runtime's definition of an event.
```

Comment:
be careful what happens to spacing. I think we follow that everything uses tabs

Reply:
Good to know!